lib/rules/selector-type-no-unknown/__tests__/index.js

```diff
@@ -50,6 +50,16 @@ testRule(rule, {
   }, {
     code: "a:nth-child(n) {}",
   }, {
+    code: "input:not(*) {}",
```

Comment:
I'm not sure what elements will be targeted by this selector. Do you have idea? :) It targets `input`, which should not be targeted by `*`. But every `input` is targeted by `*`.